tests/test_input.py

```diff
@@ -12,22 +15,65 @@ def test_input_valid_working_dir_():
     # Assert path to DICOMs exists
     assert valid_filesystem_input.working_directory.is_dir()
 
+    valid_tcia_input = TCIAInput(collection="", patient_ids=[], modalities=[])
+    # Assert path to DICOMs exists
+    assert valid_tcia_input.working_directory.is_dir()
+
+
+def assert_invalid_tcia_input(invalid_tcia_input):
+    """
+    Assert path to DICOMs does exist, but it contains no files
+    """
+    invalid_tcia_input.fetch_data()
+    assert invalid_tcia_input.working_directory.is_dir()
+    assert (
+        len(
+            [
+                name
+                for name in os.listdir(invalid_tcia_input.working_directory)
+                if os.path.isfile(name)
+            ]
+        )
+        == 0
+    )
+
 
-def test_input_invalid_working_dir_():
-    invalid_test_input = WebInput(working_directory="NOT_VALID_PATH", data_url="")
+def test_input_invalid_working_dir():
+    invalid_test_input = WebInput(data_url="", working_directory="INVALID_PATH")
     # Assert path to DICOMs does not exist
     assert not invalid_test_input.working_directory.is_dir()
 
-    invalid_filesystem_input = FilesystemInput(working_directory="NOT_VALID_PATH")
+    invalid_filesystem_input = FilesystemInput(working_directory="INVALID_PATH")
     # Assert path to DICOMs does not exist
     assert not invalid_filesystem_input.working_directory.is_dir()
 
+    invalid_work_dir_tcia_input = TCIAInput(
```

Comment:
Might just need to chang the name of this variable `invalid_work_dir_tcia_input`

Reply:
Hmmm wait, I think I misunderstood... Never mind this is fine